src/Grpc.AspNetCore.Server.Reflection/GrpcReflectionServiceExtensions.cs

```diff
@@ -120,6 +121,7 @@ public static IServiceCollection AddGrpcReflection(this IServiceCollection servi
             return null;
         }
 
+        [SuppressMessage("Trimming", "IL2075", Justification = "Modern apps published with trimming will be using newer Grpc.Tools and won't use this fallback.")]
```

Comment:
Is this guaranteed and/or are the situations where it happens very edge-casey?

Reply:
Edge-casey. Apps won't use this path unless they use a Grpc.Tools version that's 2+ years old.